raft/quorum/testdata/joint_commit.txt

```diff
@@ -0,0 +1,476 @@
+## No difference between a simple majority quorum and a simple majority quorum
```

Comment:
> How about AckIndexer? I also found that from far away IndexLookuper didn't impart much intuition about the importance of the interface and why it was being used.

Can't reply to the original comment for some reason but here we go:

Honestly I can't relate to AckIndexer much at all. What's the ack? Why is an ack being indexed? I'm not going to say that I love `IndexLookuper` but at least it tells me that it "looks up an index" which hopefully suggests that you put $something in and get an "index" out.

Since I want to play with rafttoy before merging this, let's bikeshed some more... `VoterToIndexMapper`? Doesn't quite roll of the tongue but hopefully *actually* explains it.